meson.build

```diff
@@ -40,15 +43,83 @@ postgresql = declare_dependency(
   ],
 )
 
+pgrn_dependencies = [postgresql, groonga, libm]
+
+pgrn_cpp_args = []
+if get_option('msgpack').enabled()
+  msgpack = dependency(get_option('msgpack_package_name'), required: true)
+  pgrn_dependencies += msgpack
+  pgrn_cpp_args += '-DPGRN_HAVE_MSGPACK'
```

Comment:
We should add a dependency for this: `pgrn_msgpack = declare_dependency(compile_args: ['-DPGRN_HAVE_MSGPACK'], dependencies: [msgpack])`